### DISCUSSION TITLE
Sailfish OS in Mi 9T Pro

### DISCUSSION THREAD

**Discussion by thetechgeekster** [sailfish-on-davinci/issues#18] [Q&A]
Hi!
I would like to start porting Sailfish OS 4 in Mi 9T Pro. As far as I could see, I should use the Lineage OS 16 as base. Could the dev working for the Mi 9T port "guide" me into the murky waters of the porting procedure? The official guide is a bit... messy. 

Thank you very much!

**Comment by 0312birdzhang** [comment]
You can take a look https://github.com/Sailfish-OS-on-Raphael
I'm not sure is it working or not, mi 9t is different from mi9t pro.

**Reply by thetechgeekster** [reply]
Oh ok. I'll try! Thank you!
